packages/mui-codemod/src/v6.0.0/sx-prop/test-cases/sx-css-vars.expected.js

```diff
@@ -0,0 +1,69 @@
+<Box
+  sx={theme => ({
+    backgroundImage: "var(--items-imageDark)",
+    ...theme.applyStyles("light", {
+      backgroundImage: "var(--items-imageLight)"
+    })
+  })}
+  style={{
```

Comment:
Could we add the CSS vars in the `sx` return? OR is that not supported? I mean:

```js
  sx={theme => ({
    "--items-imageLight": items[selectedItemIndex].imageLight,
    "--items-imageDark": items[selectedItemIndex].imageDark
    backgroundImage: "var(--items-imageDark)",
    ...theme.applyStyles("light", {
      backgroundImage: "var(--items-imageLight)"
    })
  })}
```

Reply:
That's not possible for Pigment because `"--items-imageLight": items[selectedItemIndex].imageLight` is dynamic. The reason for using CSS variables is to move the dynamic part to React inline styles while the CSS is static (does not change between renders) because the CSS variable name does not change.